File: app/src/main/java/com/syber/ssspltd/response/BannerResponse/BannerPojo.java

package com.syber.ssspltd.response.BannerResponse;

import java.util.List;
import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class BannerPojo {

    @SerializedName("BannerListResult")
    private List<BannerListResult> mBannerListResult;
    @SerializedName("ResponseCode")
    private Long mResponseCode;
    @SerializedName("ResponseMessage")
    private String mResponseMessage;
    @SerializedName("ResponseStatus")
    private Boolean mResponseStatus;

    public List<BannerListResult> getBannerListResult() {
        return mBannerListResult;
    }

    public void setBannerListResult(List<BannerListResult> bannerListResult) {
        mBannerListResult = bannerListResult;
    }

    public Long getResponseCode() {
        return mResponseCode;
    }

    public void setResponseCode(Long responseCode) {
        mResponseCode = responseCode;
    }

    public String getResponseMessage() {
        return mResponseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        mResponseMessage = responseMessage;
    }

    public Boolean getResponseStatus() {
        return mResponseStatus;
    }

    public void setResponseStatus(Boolean responseStatus) {
        mResponseStatus = responseStatus;
    }

}

File: app/src/main/java/com/syber/ssspltd/data/model/BannerPojo.kt
package com.syber.ssspltd.data.model

import com.google.gson.annotations.SerializedName

data class BannerPojo(
    @SerializedName("BannerList")
    val bannerList: List<BannerList>? = null,

    @SerializedName("ResponseCode")
    val responseCode: Long? = null,

    @SerializedName("ResponseMessage")
    val responseMessage: String? = null,

    @SerializedName("ResponseStatus")
    val responseStatus: Boolean? = null
)
